src/js/menu/acquia_lift.personalize.views.js

```diff
@@ -1436,11 +1119,6 @@
       this.listenTo(this.model, 'change:isActive', this.onEditModeChange);
       this.listenTo(this.campaignCollection, 'change:isActive', this.onCampaignChange);
 
-      this.onPageVariationEditModeProxy = $.proxy(this.onPageVariationEditMode, this);
```

Comment:
How is this event being triggered? I couldn't find it any more and I am still having trouble doing so haha.